lib/jsonapi_parameters/translator.rb

```diff
@@ -1,16 +1,20 @@
 module JsonApi::Parameters
-  def jsonapify(params)
-    jsonapi_translate(params)
+  def jsonapify(params, naming_convention: :snake)
+    jsonapi_translate(params, naming_convention: naming_convention)
   end
 
   private
 
-  def jsonapi_translate(params)
+  def jsonapi_translate(params, naming_convention:)
     params = params.to_unsafe_h if params.is_a?(ActionController::Parameters)
 
     return params if params.nil? || params.empty?
 
-    @jsonapi_unsafe_hash = params.deep_symbolize_keys
+    @jsonapi_unsafe_hash = if naming_convention != :underscore
```

Comment:
Shouldn't it be: `@jsonapi_unsafe_hash = if naming_convention != :snake`? It seems that the default convention is `snake` so here we should have `snake` instead of `underscore`. Please correct me if I'm missing something.

Reply:
Good catch, you are correct!